src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -58,15 +67,20 @@
         }) }}
       {% endblock %}
 
-      {% block product_tab_stock %}
-        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/stock.html.twig', {
-          'productForm': productForm,
-        }) }}
-      {% endblock %}
-
-      {% block product_tab_combinations %}
-        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
-      {% endblock %}
+      {% if productType == 'combinations' %}
+        {% block product_tab_combinations %}
+          {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
+        {% endblock %}
+      {% else %}
+        {% block product_tab_stock %}
+          <div role="tabpanel" class="form-contenttab tab-pane" id="stock-tab">
```

Comment:
Sorry to be a pain ^^
But `Tabs/virtual.html.twig` shouldn't be a tab template but at best a Block one

The principle of a tab template is that it includes `<div role="tabpanel" class="form-contenttab tab-pane" id="stock-tab">` and inside it there are some blocks and forms

We already have such tab template with `PrestaShop/Admin/Sell/Catalog/Product/Tabs/stock.html.twig` So it should be included here in the `else` condition

And it's inside this tab template that you can integrate the stock and virtual file form